app/models/communication.rb

```diff
@@ -46,14 +49,29 @@ def self.create_with_twilio_details!(user:, appointment:, twilio_sid:, twilio_ms
 
   def communication_result
     case
-    when successful?   then COMMUNICATION_RESULTS[:successful]
-    when unsuccessful? then COMMUNICATION_RESULTS[:unsuccessful]
-    when in_progress?  then COMMUNICATION_RESULTS[:in_progress]
-    else                    COMMUNICATION_RESULTS[:unknown]
+    when successful? then
+      COMMUNICATION_RESULTS[:successful]
+    when unsuccessful? then
+      COMMUNICATION_RESULTS[:unsuccessful]
+    when in_progress? then
+      COMMUNICATION_RESULTS[:in_progress]
+    else
+      COMMUNICATION_RESULTS[:unknown]
     end
   end
 
   def attempted?
     successful? || in_progress?
   end
+
+  def anonymized_data
+    { id: hash_uuid(id),
```

Comment:
[STYLE] Let's keep the map keys style consistent:

Either,

```
    {
      id: hash_uuid(id),
      created_at: created_at,
      result: result,
      duration: duration,
      start_time: start_time,
      end_time: end_time
    }
```
or, 

```
    { id: hash_uuid(id),
      created_at: created_at,
      result: result,
      duration: duration,
      start_time: start_time,
      end_time: end_time
    }
```

Reply:
Updated.